nucypher/blockchain/eth/sol/source/threshold/IStaking.sol

```diff
@@ -0,0 +1,358 @@
+    ///         the given disabled `application`, for all authorized amount.
+    ///         Can be called by anyone.
+    function forceDecreaseAuthorization(address operator, address application)
+        external;
+
+    /// @notice Pauses the given application’s eligibility to slash stakes.
+    ///         Besides that stakers can't change authorization to the application.
+    ///         Can be called only by the Panic Button of the particular
+    ///         application. The paused application can not slash stakes until
+    ///         it is approved again by the Governance using `approveApplication`
+    ///         function. Should be used only in case of an emergency.
+    function pauseApplication(address application) external;
+
+    /// @notice Disables the given application. The disabled application can't
+    ///         slash stakers. Also stakers can't increase authorization to that
+    ///         application but can decrease without waiting by calling
+    ///         `requestAuthorizationDecrease` at any moment. Can be called only
+    ///         by the governance. The disabled application can't be approved
+    ///         again. Should be used only in case of an emergency.
+    function disableApplication(address application) external;
+
+    /// @notice Sets the Panic Button role for the given application to the
+    ///         provided address. Can only be called by the Governance. If the
+    ///         Panic Button for the given application should be disabled, the
+    ///         role address should be set to 0x0 address.
+    function setPanicButton(address application, address panicButton) external;
+
+    /// @notice Sets the maximum number of applications one operator can
+    ///         authorize. Used to protect against DoSing slashing queue.
+    ///         Can only be called by the Governance.
+    function setAuthorizationCeiling(uint256 ceiling) external;
+
+    //
+    //
+    // Stake top-up
+    //
+    //
+
+    /// @notice Increases the amount of the stake for the given operator.
+    ///         Can be called only by the owner or operator.
+    /// @dev The sender of this transaction needs to have the amount approved to
+    ///      transfer to the staking contract.
+    function topUp(address operator, uint96 amount) external;
+
+    /// @notice Propagates information about stake top-up from the legacy KEEP
+    ///         staking contract to T staking contract. Can be called only by
+    ///         the owner or operator.
+    function topUpKeep(address operator) external;
+
+    /// @notice Propagates information about stake top-up from the legacy NU
+    ///         staking contract to T staking contract. Can be called only by
+    ///         the owner or operator.
+    function topUpNu(address operator) external;
+
+    //
+    //
+    // Undelegating a stake (unstaking)
+    //
+    //
+
+    /// @notice Reduces the liquid T stake amount by the provided amount and
```

Comment:
Just checking :
*"Reduces the liquid T stake amount"* - are we referring to staked T as *"liquid" T stake"*?

Reply:
I think so, because there is no difference between KEEP/NU and T stakes except staker can withdraw T stake

Reply:
Something being liquid/unlocked vs staked was different in our old terminology, but perhaps this has changed for Threshold.

I was wondering if the comment was more like "Reduces the T stake amount..."

Reply:
yeah, this interface is just a copy from Threshold repo
Main emphasis was to distinguish stake from legacy Keep/Nu and from liquid T